services/ui-src/index.html

```diff
@@ -1,64 +1,65 @@
+        Unlike "/favicon.ico" or "favicon.ico", "/favicon.ico" will
+        work correctly both with client-side routing and a non-root public URL.
+        Learn how to configure a non-root public URL by running `npm run build`.
+      -->
+    <title>CHIP Statistical Enrollment Data Reports</title>
+    <script src="/env-config.js"></script>
+    <link rel="stylesheet" type="text/css"
+      href="https://fonts.googleapis.com/css?family=PT+Serif|Open+Sans:300,400,600,700,800" />
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" />
+    <!-- Tealium Analytics Tag Manager -->
+    <script type="text/javascript">
+      window.utag_cfg_ovrd = { noview: true };
+      const tealiumEnvMap = {
+        "mdctseds.cms.gov": "prod",
+        "mdctsedsval.cms.gov": "qa"
+      };
+      const tealiumEnv = tealiumEnvMap[window.location.hostname] || "dev";
 
-      Unlike "/favicon.ico" or "favicon.ico", "/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-  <title>CHIP Statistical Enrollment Data Reports</title>
-  <script src="/env-config.js"></script>
-  <link rel="stylesheet" type="text/css"
-    href="https://fonts.googleapis.com/css?family=PT+Serif|Open+Sans:300,400,600,700,800" />
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" />
-  <!-- Tealium Analytics Tag Manager -->
-  <script>
-    window.utag_cfg_ovrd = { noview: true };
-    const tealiumEnvMap = {
-      "mdctseds.cms.gov": "prod",
-      "mdctsedsval.cms.gov": "qa"
-    };
-    const tealiumEnv = tealiumEnvMap[window.location.hostname] || "dev";
-    const tealiumDomainMap = {
-      "prod": "tags.tiqcdn.com",
-      "qa": "tags.tiqcdn.com",
-      "dev": "tealium-tags.cms.gov",
-    }
-    const tealiumDomain = tealiumDomainMap[tealiumEnv];
-    const tealiumPath = `https://${tealiumDomain}/cms-general/${tealiumEnv}`;
-    document.write(`<script src="${tealiumPath}/utag.js" async><\/script>`);
-    document.write(`<script src="${tealiumPath}/utag.sync.js"><\/script>`);
-  </script>
-</head>
+      // Load utag.sync.js
+      (function () {
+        const script = document.createElement("script");
+        script.type = "text/javascript";
+        script.src = `https://tealium-tags.cms.gov/cms-general/${tealiumEnv}/utag.sync.js`;
+        document.head.appendChild(script);
+      })();
+    </script>
+  </head>
+  <body>
+    <script type="text/javascript">
+      // Load utag.js
+      (function (a, b, c, d) {
+        a = `https://tealium-tags.cms.gov/cms-general/${tealiumEnv}/utag.js`;
+        b = document;
+        c = "script";
+        d = b.createElement(c);
+        d.src = a;
+        d.type = "text/javascript";
+        d.async = true;
+        a = b.getElementsByTagName(c)[0];
+        a.parentNode.insertBefore(d, a);
+      })();
```

Comment:
Not a suggestion! But I need to complain: it kills me to see all this obfuscation that really could & should just be one line, on any modern browser:

```suggestion
      document.head.append(`<script async src="https://tealium-tags.cms.gov/cms-general/${tealiumEnv}/utag.js"></script>`);
```

But I guess the folks at CMS/tealium really want to make sure the analytics can run even on ~decade-old browsers~... ~"security solutions"~... ~piles of trash~ Internet Explorer, which never supported using `append` that way. If this passes our testing, then so be it.

Reply:
Is it really just for internet explorer?

Reply:
Yep. There are a few ways to add elements to the DOM, like `.appendChild()`, `.insertBefore()`, `.append()`, and some others. One of the advantages of `.append()` is that [it supports strings](https://developer.mozilla.org/en-US/docs/Web/API/Element/append), so that you don't have to instantiate your new element ahead of time with `document.createElement()`.

And [support for `append()` started really spreading in 2016](https://developer.mozilla.org/en-US/docs/Web/API/Element/append), and has been all but guaranteed since 2018. In _real_ browsers, that is. You can see the red column for IE 11 there, which was released in 2013. Not that it would have been likely to have support for this, even if it was supported by other browsers at the time. IE was never famous for its compatibility with web standards.